src/typescript-transpile.js

```diff
@@ -0,0 +1,152 @@
+// From https://github.com/teppeis/typescript-simple/blob/master/index.js
```

Comment:
Would you mind converting this file to CoffeeScript with a 2-space indent? We use CoffeeScript in this repository when possible.

If you'd like to keep it as JavaScript then I'd suggest extracting it to a library, publishing it to npm, and then just using it as a library here.